src/dapp/DSRuneAbstract.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.12;
+
+// https://github.com/makerdao/dss-spellbook
+contract DSRuneAbstract {
```

Comment:
Suggestion: whatever the name ends up being, add a comment to each function describing what it is/does/etc.

Reply:
Agreed.